frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -208,6 +208,16 @@ export const StyledSidebarNavLink = styled.a<StyledSidebarNavLinkProps>(
 export const StyledSidebarLinkText = styled.span<StyledSidebarNavLinkProps>(
   ({ isActive, theme }) => ({
     color: isActive ? theme.colors.bodyText : theme.colors.fadedText60,
+
+    // We only want it to to be truncated on desktop,
+    // where we can use the tooltip to show the whole content.
```

Comment:
> where we can use the tooltip to show the whole content.

but with this PR there isn't any tooltip anymore, or?

Reply:
True! Removed the media query so it wraps everywhere in [4099d47](https://github.com/streamlit/streamlit/pull/5538/commits/4099d471c8dabbf07db64d255959dd2c120db628)